### PR TITLE
Fix 14802 - Prevent children error in page container

### LINKED ISSUE #14802: [Bug]: Cannot read properties of undefined (reading 'children')
### Describe the bug

**Problem**: when I'm in the process of importing a Mainnet custom token, and I change the network, I land into an error page with the message `Cannot read properties of undefined (reading 'children')`

https://user-images.githubusercontent.com/54408225/171133656-d8caea5a-3f9c-4518-b134-815900761f4e.mp4

### Steps to reproduce

1. Unlock Metamask
2. Select Mainnet Network
3. Click Import Tokens
4. Click Custom Token
5. Introduce a token address (i.e. `0xB8c77482e45F1F44dE1745F52C74426C631bDD52`)
6. Change the Network to Rinkeby

### Error messages or log output

```shell
Cannot read properties of undefined (reading 'children')
```

### Version

10.14.6

### Build type

_No response_

### Browser

Chrome

### Operating system

Linux

### Hardware wallet

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by darkwing**
## Explanation

This error occurs because the `activeTabIndex` is `1`; when switching to another network, we remove the first tab, and thus `children[activeTabIndex]` doesn't exist and thus bombs the extension UI.

## More Information

* Fixes [#12345](https://github.com/MetaMask/metamask-extension/issues/14802)

## Manual Testing Steps

1.  Unlock Metamask
2.  Select Mainnet Network
3.  Click Import Tokens
4.  Click Custom Token
5.  Introduce a token address (i.e. `0xB8c77482e45F1F44dE1745F52C74426C631bDD52`)
6.  Change the Network to Rinkeby -- notice the page looking correct
7.  Switch back to Mainnet -- notice the page looking correct

## Pre-Merge Checklist

- [ ] PR template is filled out
- [ ] Manual testing complete & passed
- [ ] **IF** this PR fixes a bug, a test that _would have_ caught the bug has been added
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** QA attention is required, "QA Board" label has been applied
- [ ] PR has been added to the appropriate release Milestone

### CODE REVIEW

File: ui/components/ui/page-container/page-container.component.js [hunk 1]
```diff
@@ -74,9 +74,11 @@ export default class PageContainer extends PureComponent {
     children = children.filter(Boolean);
     const { activeTabIndex } = this.state;
 
-    return children[activeTabIndex]
-      ? children[activeTabIndex].props.children
-      : children.props.children;
+    if (Array.isArray(children)) {
```
Comment (segun) [inline ui/components/ui/page-container/page-container.component.js:77]: I think line 74 makes this check redundant.

Comment (darkwing) [reply ui/components/ui/page-container/page-container.component.js:77]: Good shout, will look again.

**Review by digiwand — APPROVED** [review]
Nice. Verified the fix works 👍

**Review by adonesky1 — APPROVED** [review]
LGTM! Also verified.

**Pull request merged**